packages/useValidators/test.js

```diff
@@ -0,0 +1,57 @@
+const React = require('react');
+const { useContext, createElement: h } = require('react');
+const { render, cleanup, fireEvent } = require('react-testing-library');
+
+const useValidators = require('./');
+
+function TestApp({ validators }) {
+  const [ValidationContext, invokeValidators] = useValidators(validators);
+  const results = useContext(ValidationContext);
+
+  function onChange(event) {
+    invokeValidators(event.target.value);
+  }
+
+  return h('div', null, [
+    h('label', { htmlFor: 'test' }, 'Test'),
+    h('input', { type: 'text', id: 'test', onChange }),
+    results.map((res, index) => (
+      res.isValid ? 
+      h('p', { key: index }, 'Valid') :
+      h('p', { key: index }, 'Invalid')
+    ))
+  ]);
```

Comment:
For those who haven't seen what JSX looks like after it gets transformed seeing all these `h` calls can be very confusing. Would it be worth it to provide a `jsx` equivalent in comments just above this return?

Reply:
done.